### ISSUE TITLE
Exposing the result as yaml

### ISSUE THREAD

**Issue by baloo** [awakesecurity/terraform-provider-dhall#3]
It could be worth exposing the result of the evaluation as YAML:
```terraform
# resources.tf
resource "aws_iam_policy" "assets-access" {
  name  = "assets-access"
  description = "A policy allowing access to assets."

  policy = data.dhall.foo.yaml
}
````

Exporting a new value here should suffice
https://github.com/awakesecurity/terraform-provider-dhall/blob/914e3ff64f858596b23ea4eacbe0ef50a9523ab0/internal/provider/data_source.go#L77-L81

**Comment by isaacsanders** [comment]
couldn't one just take the JSON and re-encode it as YAML?

**Comment by baloo** [comment]
You mean using a helper in terraform? ([jsondecode](https://www.terraform.io/docs/language/functions/jsondecode.html) and [yamlencode](https://www.terraform.io/docs/language/functions/yamlencode.html))
I thought it could have been a bit more useful/less verbose to do that in the go plugin. But otherwise yes, decode json then reencode it in yaml in golang.

**Comment by isaacsanders** [comment]
Yeah, the terraform helper. I would think it would be nice to keep the dhall part of the tooling as simple as possible.
